specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -340,8 +332,6 @@ namespace Microsoft.EventGrid {
 
   // RenewLock Operation:
   // POST https://{namespaceName}.{region}.eventgrid.azure.net/topics/{topicName}/eventsubscriptions/{eventSubscriptionName}:renewLock?api-version={apiVersion}
-  @added(ServiceApiVersions.v2023_10_01_preview)
-  @removed(ServiceApiVersions.v2023_11_01)
```

Comment:
can remove this since we have the @added below

Reply:
I'm glad, it was horrible to have this added/removed/added